tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml

```diff
@@ -318,3 +319,48 @@ jobs:
     displayName: 'Run C++ BERT-L convergence test'
     retryCountOnTaskFailure: 1
     condition: and(succeededOrFailed(), eq(variables.onnxruntimeBuildSucceeded, 'true')) # ensure all tests are run when the build successed
+
+  - script: |
+      sudo apt-get update
+      sudo apt install -y cifs-utils
+    displayName: 'Install filesystems util'
+
+  - bash: tools/ci_build/github/linux/docker/scripts/training/azure_scale_set_vm_mount_test_data.sh -p $(orttrainingtestdatascus-storage-key) -s "//orttrainingtestdatascus.file.core.windows.net/mnist" -d "/mnist"
+    displayName: 'Mount MNIST'
+    condition: succeededOrFailed()
+
+  - bash: tools/ci_build/github/linux/docker/scripts/training/azure_scale_set_vm_mount_test_data.sh -p $(orttrainingtestdatascus-storage-key) -s "//orttrainingtestdatascus.file.core.windows.net/bert-data" -d "/bert_data"
+    displayName: 'Mount bert-data'
+    condition: succeededOrFailed()
+
+  - bash: tools/ci_build/github/linux/docker/scripts/training/azure_scale_set_vm_mount_test_data.sh -p $(orttrainingtestdatascus-storage-key) -s "//orttrainingtestdatascus.file.core.windows.net/hf-models-cache" -d "/hf_models_cache"
+    displayName: 'Mount hf-models-cache'
+    condition: succeededOrFailed()
+
+
+  # Entry point for all ORTModule tests
+  # The onnxruntime folder is deleted in the build directory
+  # to enforce use of the onnxruntime wheel
+  # Uninstall torch 1.10 from the docker image above and install torch 1.11 before running tests.
+  #
+  # At this point, the best whl available with torch 1.11 (which supports autograd) is located in https://repo.radeon.com/rocm/manylinux/rocm-rel-5.1.3/ 
+  # which is a release for internal use. This release will be available via Torch's download page. We can either continue 
+  # to use the link below, or switch to the link from Torch's when it's published. Ideally, the AMD CI UTs above could also migrate to use
+  # the same whl below.   
+  - task: CmdLine@2
+    inputs:
+      script: |-
+        cd ./build/RelWithDebInfo
+        unset PYTHONPATH
+        pip uninstall -y torch torchvision
+        pip install torch==1.11.0 torchvision==0.12.0 -f https://repo.radeon.com/rocm/manylinux/rocm-rel-5.1.3/
+        pip install wget pytorch_lightning==1.6.0
```

Comment:
Shall we put this into a requirements.txt file?

Reply:
> Shall we put this into a requirements.txt file?

We could. However, we need to create new requirements.txt file, the original one would mess up with the env. Since these are only minimum changes we need, I am making it explicit for now. There's ROCm 5.2 just released, I am thinking of having a separate PR to create the new requirements.txt and new docker image for AMD CI (and possibly deprecate older config we don't need to support, e.g., 4.3.1/torch10.x, etc. for packaging CI) when 5.2 is brought in. What do you think? (i.e., I can still create new requirements.txt in this PR)